src/app/components/bibliography/bibliography.component.html

```diff
@@ -0,0 +1,19 @@
+<div class='biblSpace'>
+  <div class='biblContent'>
+    <span>
+      <span *ngIf="isComplete" class="biblSortingOption">
+        <label> Sort By </label>
+        <ng-select class="w-25 p-3" placeholder="Select" [items]="(sortingField | async)?.sortBy" [clearable]="false"
+          [(ngModel)]="selectSortByField" (change)="sortBy(selectSortByField)" [searchable]="false">
+        </ng-select>
+      </span>
+      <span class="biblSortingDefault">
+        <label class="pr-1 selectLabel"> Alphabetical order</label>
```

Comment:
I would like to. Where can I get the selected language? I checked `app.config` and `main-menu`. In my opinion this could be the method in which the selected language revolves around:

```
// LANGUAGE
selectLanguage(event: MouseEvent, languageSelected: Language) {
    event.stopPropagation();
    this.translate.use(languageSelected.code);
    this.itemClicked.emit('language');
}
```
Quite right?